utils/R/read.output.R

```diff
@@ -87,10 +87,11 @@ model2netcdf <- function(runid, outdir, model, lat, lon, start_date, end_date) {
 ##' @param start.year first year of output to read (should be greater than ) 
 ##' @param end.year last year of output to read
 ##' @param variables variables to be read from model output
+##' @param dataframe A boolean that will return output in a data.frame format with posix column. Usefull for align.data and plotting. 
```

Comment:
Ok, so this isn't as bad of a hack as I expected -- I suspect this should be the default (dataframe=TRUE) with the false flag retained as an option just for backward compatibility. You might want to do a quick grep of the code to see where else read.output is used and figure out if any of those will be broken by changing the default (in which case you should add an explicit dataframe=FALSE to those calls).  Since you have this option you should also eliminate the equivalent code in the benchmarking, which is now redundant.